File: Week 13 Go Multithread/Go/internal/model/task.go
package model

import "fmt"

// Task represents a unit of work to be processed
type Task struct {
	ID   int
	Data string
}

// String returns a string representation of the task
func (t Task) String() string {
	return fmt.Sprintf("Task{id=%d, data='%s'}", t.ID, t.Data)
}

File: Week 13 Multithread/Java/src/Task.java

public class Task {
    private final int id;
    private final String data;

    public Task(int id, String data) {
        this.id = id;
        this.data = data;
    }

    public int getId() {
        return id;
    }

    public String getData() {
        return data;
    }

    @Override
    public String toString() {
        return "Task{id=" + id + ", data='" + data + "'}";
    }
}